packages/editor/src/package/editor.tsx

```diff
@@ -40,6 +53,21 @@ export function SerloEditor(props: SerloEditorProps) {
     ...props,
   }
 
+  const initialState = !props.initialState
+    ? createEmptyDocument(editorVariant)
+    : props.initialState
+
+  const { migratedState, stateChanged } = migrate(initialState)
```

Comment:
this would run on every render, right?

Reply:
Yes. If prop `initialState` changed, we need to migrate again. If it didn't change, I think it probably does not hurt to just redo the migration.

Reply:
I learned to never trust react not to rerender, but let's see :)